lib/p2p/Pool.ts

```diff
@@ -328,6 +328,9 @@ class Pool extends EventEmitter {
    * Attempt to create an outbound connection to a node using its known listening addresses.
    */
   private tryConnectNode = async (node: NodeConnectionInfo, retryConnecting = false) => {
+    if (node.inbound) {
```

Comment:
Same here, also I think this would have the undesired affect of making it impossible to create an outbound connection to a given node, even if we try to do it manually.